scripts/rook.sh

```diff
@@ -5,14 +5,31 @@ ROOK_DEPLOY_TIMEOUT=${ROOK_DEPLOY_TIMEOUT:-300}
 ROOK_URL="https://raw.githubusercontent.com/rook/rook/${ROOK_VERSION}/cluster/examples/kubernetes/ceph"
 ROOK_BLOCK_POOL_NAME=${ROOK_BLOCK_POOL_NAME:-"newrbdpool"}
 ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
+KUBECTL_RETRY=5
+KUBECTL_RETRY_DELAY=10
 
 rook_version() {
 	echo "${ROOK_VERSION#v}" | cut -d'.' -f"${1}"
 }
 
+kubectl_retry() {
+    local retries=0
+
+    while ! kubectl "${@}"
+    do
+        retries=$[retries + 1]
+        if [ ${retries} -eq ${KUBECTL_RETRY} ]
+        then
+            return 1
```

Comment:
exit or return is the best option?

Reply:
return from the function, have it behave similar to the return value of `kubectl`